rsconf/package_data/dev/db/000.yml.jinja

```diff
@@ -81,6 +81,7 @@ default:
     on_calendar: "6:30"
   docker:
     iptables: false
+    min_version: 23.0.0
```

Comment:
min_version is not descriptive enough. min_software_version is clearer.

Reply:
0349609